custom/logistics/api.py

```diff
@@ -129,6 +133,15 @@ def save_custom_fields(self, definition_name, custom_fields):
             if need_save:
                 fields_definitions.save()
 
+    @memoized
+    def _get_logistics_domains(self):
+        return ILSGatewayConfig.get_all_enabled_domains() + EWSGhanaConfig.get_all_enabled_domains()
+
+    def set_default_backend(self):
+        domain_object = Domain.get_by_name(self.domain)
+        domain_object.default_sms_backend_id = "MOBILE_BACKEND_TEST"
```

Comment:
This should be set to:

    MobileBackend.load_by_name(None, 'MOBILE_BACKEND_TEST')._id

(corehq.apps.sms.mixin import MobileBackend)